package.json

```diff
@@ -72,7 +72,7 @@
     "@uniswap/router-sdk": "^1.4.0",
     "@uniswap/sdk-core": "^4.0.7",
     "@uniswap/smart-order-router": "3.17.1",
-    "@uniswap/uniswapx-sdk": "1.5.0-alpha.7",
+    "@uniswap/uniswapx-sdk": "../gouda-sdk",
```

Comment:
revert

Reply:
yep will update with newest release once we release it